README.md

```diff
@@ -27,40 +28,43 @@ $ go get -u github.com/minio/minio-go
 ```
 ## Initialize Minio Client
 
-You need four items in order to connect to Minio object storage server.
+You need four items to connect to Minio object storage server.
 
 
 
-| Params  | Description| 
+| Parameter  | Description| 
 | :---         |     :---     |
 | endpoint   | URL to object storage service.   | 
-| accessKeyID | Access key is like user ID that uniquely identifies your account. |   
+| accessKeyID | Access key is the user ID that uniquely identifies your account. |   
 | secretAccessKey | Secret key is the password to your account. |
-|secure | Set this value to 'true' to enable secure (HTTPS) access. |
+| secure | Set this value to 'true' to enable secure (HTTPS) access. |
 
 
 ```go
 
 package main
 
 import (
-  "fmt"
-
-  "github.com/minio/minio-go"
+    "fmt"
+    "github.com/minio/minio-go"
 )
 
 func main() {
-        // Use a secure connection.
-        ssl := true
-
-        // Initialize minio client object.
-        minioClient, err := minio.New("play.minio.io:9000", "Q3AM3UQ867SPQQA43P2F", "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG", ssl)
-        if err != nil {
-                fmt.Println(err)
-                return
-        }
+    endpoint := "play.minio.io:9000"
```

Comment:
Previous tabbing is the correct tabbing, you need to use 'gofmt -s' to see the final tabbing for example here

```
$ gofmt -d test.go 
diff test.go gofmt/test.go
--- /tmp/gofmt162682432	2016-08-13 19:31:34.427524804 -0700
+++ /tmp/gofmt933985951	2016-08-13 19:31:34.427524804 -0700
@@ -1,22 +1,21 @@
 package main
 
 import (
-    "fmt"
-    "github.com/minio/minio-go"
+	"fmt"
+	"github.com/minio/minio-go"
 )
 
 func main() {
-    endpoint := "play.minio.io:9000"
-    accessKeyID := "Q3AM3UQ867SPQQA43P2F"
-    secretAccessKey := "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG"
-    useSSL := true
+	endpoint := "play.minio.io:9000"
+	accessKeyID := "Q3AM3UQ867SPQQA43P2F"
+	secretAccessKey := "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG"
+	useSSL := true
 
-    minioClient, err := minio.New(endpoint, accessKeyID, secretAccessKey, useSSL)
-    if err != nil {
-        fmt.Println(err)
-        return
-    }
+	minioClient, err := minio.New(endpoint, accessKeyID, secretAccessKey, useSSL)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
 
-    fmt.Println("%v", minioClient)          // minioClient is now setup
+	fmt.Println("%v", minioClient) // minioClient is now setup
 }
-
```